web/src/components/PoiListItem.tsx

```diff
@@ -1,17 +1,73 @@
 import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
 
-import { PoiModel } from 'api-client'
+import { GeoJsonPoiProperties } from 'api-client'
 
-import ListItem from './ListItem'
+import PoiPlaceholder from '../assets/PoiPlaceholderThumbnail.jpg'
+import CleanLink from './CleanLink'
+
+const ListItemContainer = styled.article`
+  display: flex;
+  border-bottom: 1px solid ${props => props.theme.colors.textSecondaryColor};
+  padding: 22px 0;
+
+  &:first-child {
+    padding-top: 0;
+  }
+`
+
+const Thumbnail = styled.img`
+  width: 100px;
+  height: 100px;
+  flex-shrink: 0;
+  border: 1px solid transparent;
```

Comment:
What is a transparent border good for?

Reply:
you can not get rounded corners for pictures without a border

Reply:
yes you can by adding border-radius?